modules/update-cluster-version-object.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-cluster-version-object_{context}"]
+= The ClusterVersion object
+
+One of the resources that the Cluster Version Operator (CVO) monitors is `ClusterVersion`.
+
+Administrators and OpenShift components can communicate or interact with the CVO through the `ClusterVersion` object.
+The desired CVO state is declared through the `ClusterVersion` object and the current CVO state is reflected in the object's status.
+
+[NOTE]
+====
+Do not directly modify the `ClusterVersion` object. Instead, use interfaces such as the `oc` CLI or the web console to declare your desired update target.
+====
+
+The CVO continually reconciles the cluster with the target state declared in `ClusterVersion` `spec`.
+When the desired release differs from the current one, that reconciliation updates the cluster.
```

Comment:
desired -> required

Reply:
"desired" in this instance doesn't refer to user intention, but rather the desired state of the cluster, where the reconciliation process tries to make the actual state match the desired state. So when the desired version is different from the actual version, the reconciliation to match the desired state is the actual cluster update process we're documenting here.

I can reword to make this more clear, but I think keeping "desired" in here is important for describing how the process works, despite the ISG guidance. Let me know what you think!

Reply:
Ahh I get what you're saying, yeah I think that the reword would be great!